### ISSUE TITLE
I'm having an error running process

### ISSUE THREAD

**Issue by DLPDarkNova** [kohya-ss/sd-webui-additional-networks#24]
Can you please help, I encountered an error like this when I was generating:

```
LoRA weight: 1, model: Mix2R4 (3a125b2e)
dimension: 1000, multiplier: 1
create LoRA for Text Encoder: 72 modules.
create LoRA for U-Net: 192 modules.
Error running process: C:\Stable Diffusion\stable-diffusion-webui\extensions\sd-webui-additional-networks\scripts\additional_networks.py
Traceback (most recent call last):
  File "C:\Stable Diffusion\stable-diffusion-webui\modules\scripts.py", line 338, in process
    script.process(p, *script_args)
  File "C:\Stable Diffusion\stable-diffusion-webui\extensions\sd-webui-additional-networks\scripts\additional_networks.py", line 177, in process
    network, info = lora_compvis.create_network_and_apply_compvis(du_state_dict, weight, text_encoder, unet)
  File "C:\Stable Diffusion\stable-diffusion-webui\extensions\sd-webui-additional-networks\scripts\lora_compvis.py", line 78, in create_network_and_apply_compvis
    state_dict = network.apply_lora_modules(du_state_dict)              # some weights are applied to text encoder
  File "C:\Stable Diffusion\stable-diffusion-webui\extensions\sd-webui-additional-networks\scripts\lora_compvis.py", line 246, in apply_lora_modules
    state_dict = LoRANetworkCompvis.convert_state_dict_name_to_compvis(self.v2, du_state_dict)
  File "C:\Stable Diffusion\stable-diffusion-webui\extensions\sd-webui-additional-networks\scripts\lora_compvis.py", line 152, in convert_state_dict_name_to_compvis
    compvis_name = LoRANetworkCompvis.convert_diffusers_name_to_compvis(v2, tokens[0])
  File "C:\Stable Diffusion\stable-diffusion-webui\extensions\sd-webui-additional-networks\scripts\lora_compvis.py", line 141, in convert_diffusers_name_to_compvis
    assert cv_name is not None, f"conversion failed: {du_name}"
AssertionError: conversion failed: betas
```

The result was the same as when I wasn't using lora.

**Comment by kohya-ss** [comment]
Hi, there is an issue for the LoRA model with an extremely large dimensions. I will fix the issue in near future.

Please use under 256 or 128 dims.

**Comment by kohya-ss** [comment]
I think this issue is resolved by #51 . If you have any problem, please re-open the issue.
